pkg/plugins/runtime/k8s/controllers/pod_converter_test.go

```diff
@@ -378,7 +375,8 @@ var _ = Describe("InboundTagsForService(..)", func() {
 			// given
 			pod := &kube_core.Pod{
 				ObjectMeta: kube_meta.ObjectMeta{
-					Labels: given.podLabels,
+					Namespace: "test",
```

Comment:
Would this test be less confusing in the namespace of the service and the namespace of pods would match?